src/terrain/services/permanent_id_requests.clj

```diff
@@ -43,18 +44,25 @@ For example, https://doi.org/10.7946/P2G596 links to the DOI 10.7946/P2G596.")
   [path]
   (str (ft/rm-last-slash (config/permanent-id-target-base-url)) (format-publish-path path)))
 
+(defn- find-attr-value
+  [avus attr]
+  (->> avus
+       (filter #(= attr (:attr %)))
+       first
+       :value))
+
 (defn- validate-ezid-metadata
```

Comment:
I considered it, but since we're still using DataCite's EZID API for an easy transition for now, I figured we'd keep the references to `ezid` in the function names. I'll review the error messages to see where it makes sense to update those, though 👍